install_compatibility_layer.sh

```diff
@@ -10,7 +10,7 @@ REPOSITORY="software.eessi.io"
 RESUME=
 RETAIN_TMP=0
 STORAGE=
-VERSION=2023.06
+VERSION=
```

Comment:
This change essentially makes using the `--version` option mandatory?

Should we add a check to make sure that `$VERSION` is non-empty before we start using it, to avoid surprises?

Reply:
The check was already part of the script:
https://github.com/EESSI/compatibility-layer/blob/main/install_compatibility_layer.sh#L175
I.e. it passes `-e eessi_version=$VERSION` to Ansible only if it `$VERSION` is set. If it's not, Ansible will just use the default value from https://github.com/EESSI/compatibility-layer/blob/main/ansible/playbooks/roles/compatibility_layer/defaults/main.yml#L3. So in some sense it was quite useless to also have a default value set in this script.

Reply:
OK, thanks for clarifying!